components/hyper_serde/lib.rs

```diff
@@ -622,22 +623,26 @@ impl<'de> Deserialize<'de> for De<Tm> {
             where
                 E: de::Error,
             {
-                strptime(v, "%Y-%m-%dT%H:%M:%SZ")
-                    .map(De::new)
+                NaiveDateTime::parse_from_str(v, "%Y-%m-%dT%H:%M:%SZ")
+                    .map(|t| {
+                        let date_time = Utc.from_utc_datetime(&t);
+                        De::new(SystemTime::from(date_time))
+                    })
```

Comment:
Even if you don't use the upstream serde deserializer, couldn't this use https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.parse_from_rfc3339?

Reply:
I used `parse_from_rfc3339`